programs/ssl/ssl_server2.c

```diff
@@ -292,7 +294,7 @@ int main( void )
     "    renegotiation=%%d    default: 0 (disabled)\n"      \
     "    renegotiate=%%d      default: 0 (disabled)\n"      \
     "    renego_delay=%%d     default: -2 (library default)\n" \
-    "    renego_period=%%d    default: (library default)\n"
+    "    renego_period=%%d    default: (2^64 - 1)\n"
```

Comment:
It is not true if the protocol is DTLS. I suggest writing "no early renegotiation" or "maximal".